src/libcrun/seccomp.c

```diff
@@ -252,24 +252,64 @@ libcrun_generate_seccomp (libcrun_container_t *container, int outfd, unsigned in
             {
               size_t k;
               struct scmp_arg_cmp arg_cmp[6];
+              bool multiple_args = false;
+              uint32_t count[6];
+
+              memset (count, 0, sizeof (count));
+
+              for (k = 0; k < seccomp->syscalls[i]->args_len && k < 6; k++)
+                {
+                  uint32_t index;
+
+                  index = seccomp->syscalls[i]->args[k]->index;
+                  if (index >= 6)
+                    return crun_make_error (err, 0, "invalid seccomp index %zu", i);
```

Comment:
could you drop the 'z'?  `%u` should be enough.

Last thing and I think it is ready to merge

Reply:
Isn't %zu more appropriate for size_t? In theory, size_t could map to a type other than unsigned long.

Reply:
ah yes sorry.  Got confused by `index`.  Yes `%zu` is the correct type for `size_t`